### PR TITLE
Made AI medic use epipen on unconscious patients

### PR DESCRIPTION
**Pull request by LordGolias**
**When merged this pull request will:**
- Make medic AIs use epipen on unconscious patients. They will use it only after stabilizing blood loss and pain.

### CODE REVIEW

**Comment by BaerMitUmlaut** [comment]
This needs to handle revive state I believe.

**Comment by LordGolias** [comment]
@BaerMitUmlaut, why? doesn't adding time to `treatmentOverAt` forces the unit to remain in the same state, just like with morphine?

**Comment by BaerMitUmlaut** [comment]
Actually I was thinking of something else in advanced medical, my bad.

You should adjust the isInjured function to include unconsciousness though.

**Comment by LordGolias** [comment]
@BaerMitUmlaut you are right. I just updated the PR adding that.

**Comment by jonpas** [comment]
@BaerMitUmlaut Looks good?

File: addons/medical_ai/functions/fnc_healUnit.sqf [hunk 1]
```diff
@@ -82,4 +83,13 @@ switch (true) do {
             systemChat format ["%1 is giving %2 morphine", _this, _target];
         #endif
     };
+    case _needsEpinephrine: {
+        [_target] call EFUNC(medical,treatmentBasic_epipen);
```
Comment (PabstMirror) [inline addons/medical_ai/functions/fnc_healUnit.sqf]: Has no effect, should be `[_caller, _target]`

**Comment by PabstMirror** [comment]
Tested working,
I also changed `healUnit` to use the non-local versions of morphine and bandage.

**Pull request merged**